community/security/src/test/java/org/neo4j/server/security/auth/FileUserRepositoryTest.java

```diff
@@ -233,14 +239,18 @@ public void shouldIgnoreInvalidEntries() throws Throwable
         AssertableLogProvider logProvider = new AssertableLogProvider();
         Files.createDirectories( authFile.getParent() );
         Files.write( authFile, UTF8.encode(
-                "neo4j:fc4c600b43ffe4d5857b4439c35df88f:SHA-256,A42E541F276CF17036DB7818F8B09B1C229AAD52A17F69F4029617F3A554640F,FB7E8AE08A6A7C741F678AD22217808F:\n" ) );
+                "neo4j:admin:fc4c600b43ffe4d5857b4439c35df88f:SHA-256," +
```

Comment:
This has slightly wrong name/assertions/error messages. Changed the test in a new PR to reflect the actual behaviour change (we now fail hard on a wrongly formatted file).